tests/compatibility-tests/src/test/java/org/apache/activemq/artemis/tests/compatibility/JmsReplyToQueueTest.java

```diff
@@ -44,6 +46,7 @@ public static Collection getParameters() {
       combinations.add(new Object[]{SNAPSHOT, ONE_FIVE, SNAPSHOT});
       combinations.add(new Object[]{SNAPSHOT, SNAPSHOT, ONE_FIVE});
       combinations.add(new Object[]{SNAPSHOT, SNAPSHOT, SNAPSHOT});
+      combinations.add(new Object[]{JAKARTAEE, JAKARTAEE, JAKARTAEE});
```

Comment:
since you added it here, can you add the other combinations? 

it should be possible to mix producer and consumers from JAKARTA ONE_FIVE AND SNAPSHOT?

Reply:
There are 4 tests doing the same deal.. all of them should mix Jakarta and JMS on these versions IMO. it would be a good thing to do.